### PR TITLE
feat(config): add alternative device id for Heatit Z-Smoke 230V

### PR DESCRIPTION
**Pull request by roflmao**
SSIA

### CODE REVIEW

**Comment by roflmao** [comment]
@zwave-js-bot fix lint

File: packages/config/config/devices/0x019b/4512689.json [hunk 1]
```diff
@@ -10,7 +10,12 @@
 			"productType": "0x0003",
 			"productId": "0x3601",
 			"zwaveAllianceId": 3933
-		}
+		},
+    {
+      "productType": "0x0003",
+      "productId": "0x000d",
+      "zwaveAllianceId": 3933
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/4512689.json]: @blhoward2 should we remove that line if the device ID does not match the one on the alliance webite?

**Comment by AlCalzone** [comment]
@zwave-js-bot fix lint for @roflmao 🤣

**Comment by AlCalzone** [comment]
Will wait for @blhoward2's response

**Comment by blhoward2** [comment]
Please remove the zwa id from the file where the device id isn't included in the database. There is no benefit to having it included twice in the same device file and I like to keep track of which are and which aren't in the database.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**